mslice/models/labels.py

```diff
@@ -0,0 +1,32 @@
+'''Module to create axis and legend labels'''
+
+from __future__ import (absolute_import, division, print_function)
+from mslice.util import MPL_COMPAT
+
+INTENSITY_LABEL = 'Signal/#Events'
```

Comment:
If we're moving this out of the cut model then I'd suggest renaming this `CUT_INTENSITY_LABEL`